portfolio/src/main/webapp/index.html

```diff
@@ -23,8 +23,19 @@ <h1>Korhan Citlak's Portfolio</h1>
       <br>
       <p>My linkedin is <a href="https://www.linkedin.com/in/korhan-citlak-44b345191/">Linkedin</a></p>
       <br> 
-     <button onclick="getName()">Hello...</button>  
-     <div id="hello-container"></div>
-    </div>
+      <p> Comments will go down here.</p>
+    
+     <br>
+      <ul id="history"></ul>
+      <h1 id="comments"></h1>
```

Comment:
It looks like this `comments` header isn't used for anything. Consider deleting it if that's the case.